src/v1/sources/shopify/webhookTransformations/serverSideTransform.js

```diff
@@ -111,11 +113,16 @@ const processEvent = async (inputEvent, metricMetadata) => {
   // attach anonymousId if the event is track event using note_attributes
   if (message.type !== EventType.IDENTIFY) {
     await setAnonymousId(message, event, metricMetadata);
+    await updateAnonymousIdToUserIdInRedis(message);
   }
   // attach userId, email and other contextual properties
   message = handleCommonProperties(message, event, shopifyTopic);
   // add cart_token_hash to traits if cart_token is present
   message = addCartTokenHashToTraits(message, event);
+  const redisData = await RedisDB.getVal(`pixel:${message.anonymousId}`);
```

Comment:
remove `userId` handling from `handleCommonProperties` & keep it at this level along with logic of enrichment from redis data